### ISSUE TITLE
12864 LCD display 

### ISSUE THREAD

**Issue by spankurmonkey** [bigtreetech/SKR-Pico#14]
Is there a  way to map available/unused pins to connect a Mini 12864 display to the BTT SKR PICO ?

Like I saw something for the SKR E3V3 suggesting to adapt and make a Y cable to get that combination to work.

Can something similar be done on the SKR PICO?

https://github.com/bigtreetech/BIGTREETECH-SKR-mini-E3/issues/650

**Comment by yzheka** [comment]
I don't think it is possible. Instead you can buy (or maybe assemble yourself) [usb adapter board](https://a.aliexpress.com/_mL2lIUw) for 12864 display
